src/EA.Weee.RequestHandlers/AatfReturn/ObligatedSentOn/GetSentOnOperatorSiteHandler.cs

```diff
@@ -28,7 +28,12 @@ public async Task<AatfAddressData> HandleAsync(GetSentOnOperatorSite message)
 
             var aatfAddress = await getSentOnAatfSiteDataAccess.GetWeeeSentOnOperatorAddress(message.WeeeSentOnId);
```

Comment:
This class has no covering tests as far as I can see?

Reply:
I think I had previously used this request but is no longer needed, will delete.